modules/installation-configuration-parameters.adoc

```diff
@@ -254,7 +267,7 @@ endif::aws[]
 
 ifdef::osp[]
 .Additional {rh-openstack-first} parameters
-[cols=".^2m,.^3a,^3a,options="header"]
+[cols=".^2m,.^3a,^5a",options="header"]
```

Comment:
I know it was already like this, but the "Values" column is centered when all columns should be left justified.